foolbox/models/wrappers.py

```diff
@@ -81,7 +81,9 @@ def batch_predictions(self, images):
         return self.forward_model.batch_predictions(images)
 
     def predictions_and_gradient(self, image, label):
-        return self.backward_model.predictions_and_gradient(image, label)
+        predictions = self.forward_model.predictions(image)
+        gradient = self.backward_model.gradient(image, label)
+        return [predictions, gradient]
```

Comment:
should not be a list, but a tuple, and that can be returned like this (no brackets!):

```python
return predictions, gradient
```